### PR TITLE
Eliminate IonDotnet.Systems package

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: IonDotnet.Bench/WriteGround.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using BenchmarkDotNet.Attributes;
 using BenchmarkDotNet.Running;
 using IonDotnet.Serialization;
-using IonDotnet.Systems;
+using IonDotnet.Builders;
```
Comment (byronlin13) [inline IonDotnet.Bench/WriteGround.cs:4]: Can we order the imports in alphabetically?

**Comment by byronlin13** [comment]
LGTM

File: IonDotnet.Bench/WriteGround.cs (hunk 1 above)
Comment (R-maan) [reply IonDotnet.Bench/WriteGround.cs:4]: +1

Comment (ghost) [reply IonDotnet.Bench/WriteGround.cs:4]: Done.

**Comment by R-maan** [comment]
Looks good, thanks.

**Pull request closed without merging**